capellambse/model/common/element.py

```diff
@@ -199,26 +199,32 @@ def __init__(
 
         super().__init__()
         if self._xmltag is None:
-            raise TypeError(
-                f"Cannot instantiate {type(self).__name__} directly"
-            )
+            try:
+                self._xmltag = kw.pop("_xmltag")
+            except KeyError as error:
+                raise TypeError(
+                    f"Cannot instantiate {type(self).__name__} directly"
+                ) from error
```

Comment:
```suggestion
            except KeyError:
                raise TypeError(
                    f"Cannot instantiate {type(self).__name__} directly"
                ) from None
```

IMO keeping the KeyError just unnecessarily complicates the stack trace without providing additional value. Quite the opposite; it could encourage users to manually pass the `_xmltag` argument (which is definitely not what we want).